bc_obps/registration/tests/integration/test_operation_registration.py

```diff
@@ -0,0 +1,273 @@
+from datetime import timedelta
+
+import pytest
+from django.db.models import Q
+from model_bakery import baker
+from registration.models import RegistrationPurpose, FacilityDesignatedOperationTimeline, Operation
+from registration.tests.constants import MOCK_DATA_URL
+from registration.tests.utils.bakers import bc_obps_regulated_operation_baker
+from registration.tests.utils.helpers import CommonTestSetup, TestUtils
+from registration.utils import custom_reverse_lazy
+
+
+class TestOperationRegistration(CommonTestSetup):
+    def setup_method(self):
+        super().setup_method()
+        self.approved_user_operator = baker.make_recipe('utils.approved_user_operator', user=self.user)
+        self.bcghg_id = baker.make_recipe('utils.bcghg_id')
+        self.swrs_facility_id = 1003
+        self.boro_id = bc_obps_regulated_operation_baker()
+        # add some random contacts so we can test that they are not overridden
+        random_contacts = baker.make_recipe('utils.contact', _quantity=5)
+        self.operation = baker.make_recipe(
+            'utils.operation',
+            created_by=self.user,
+            operator=self.approved_user_operator.operator,
+            bcghg_id=self.bcghg_id,
+            swrs_facility_id=self.swrs_facility_id,
+            bc_obps_regulated_operation=self.boro_id,
+            contacts=random_contacts,
+        )
+        self.created_at = self.operation.created_at
+        self.updated_at = None
+        self.operation_representative_id = None
```

Comment:
I think operation rep would already be None since none of the setup appears to add it?

Reply:
Yes, it's just a variable that captures that value for later comparison. I apologize for the confusion; I've added a comment to prevent this in the future.